ios/en.lproj/InfoPlist.strings

```diff
@@ -0,0 +1,5 @@
+"NSLocationAlwaysAndWhenInUseUsageDescription" = "In addition to displaying your location on maps and planning trips, we gain valuable insights into where you and other customers travel. This helps us improve our travel offerings.";
+
+"NSLocationAlwaysUsageDescription" = "In addition to displaying your location on maps and planning trips, we gain valuable insights into where you and other customers travel. This helps us improve our travel offerings.";
```

Comment:
I feel this text is a little to "scary", especially if it is also shown to NFK/Fram which doesn't use Kogenta. I think we want the users to share their location for a better user experience, even if they don't consent to us tracking and storing their position.

Reply:
@oyvindmikkelsenatb

Reply:
I would argue that the message should be a bit "scary" since the thing that the user is sharing with us and Kogenta is important for the user to make a propper decision on. The prompt should not show for anyone that has not accepted the consent for sharing data with AtB/Kogenta. However if there is a risk that NFK/FRAM will get the same message then I think we should change it to something more generic.